test/regfox/regfox_api.test.js

```diff
@@ -1,17 +1,19 @@
-const chai = require('chai');
-const chaiAsPromised = require('chai-as-promised');
+import chai, { use, should } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
 
-chai.use(chaiAsPromised);
-chai.should();
+use(chaiAsPromised);
+should();
 const { expect } = chai;
 
-const sinon = require("sinon");
+import { stub } from "sinon";
```

Comment:
That's fair, yeah, I should have been better about not importing the entirety of sinon and chai. :)

Reply:
A _solid_ 50% of this commit is me just doing what Visual Studio Code told me to do and then trying to figure out why it did that. This was one of them, where it helpfully discovered that only the `stub` method was used and swapped out the imports. Kinda neat!